internal_ws/ykbh/src/lib.rs

```diff
@@ -348,7 +351,7 @@ impl SIRInterpreter {
                         _ => unreachable!(),
                     };
                     // Get a pointer to the return value of the called frame.
-                    let ret_ptr = oldframe.mem.local_ptr(&Local(0));
+                    let ret_ptr = oldframe.mem.local_ptr(&RETURN_LOCAL);
```

Comment:
There are other instance of `Local(0)` that presumably also need changing -- you'll need to grep the whole repo for these I suspect.

Reply:
Yeah, see https://github.com/softdevteam/yk/pull/226/files#r573834812.

My plan was to do these PRs on a per file basis, but realised now that may not be a good idea.

Reply:
I think per-file might not quite work, at least not for this particular case.

Reply:
Fixed in 8f5174b.